### DISCUSSION TITLE
在表格插槽中调用方法，方法体比重复执行

### DISCUSSION THREAD

**Discussion by lp956** [element-plus/element-plus#4819] [Q&A]
![image](https://user-images.githubusercontent.com/96097393/145926799-036734ab-73ad-40d5-bfca-9a505523e94d.png)
![image](https://user-images.githubusercontent.com/96097393/145926820-ba72c0e1-7a01-4abf-9ed9-67d2b0a8a01f.png)
重复打印了11111，有没有大佬遇到这个问题，告知一下，谢谢！！！！！

**Comment by lp956** [comment]
来个大佬分析解决一下，跪谢！！！！！！！！！！！！！！！！！！！！！！！

**Comment by lp956** [comment]
111

**Comment by emojiiii** [comment]
Can you create a issue and provide mini repo so that we can better troubleshoot the problem ？
